xbmc/Application.h

```diff
@@ -43,38 +43,37 @@ class CFileItem;
 class CFileItemList;
 class CKey;
 
-
 namespace ADDON
 {
-  class CSkinInfo;
-  class IAddon;
-  typedef std::shared_ptr<IAddon> AddonPtr;
-}
+class CSkinInfo;
+class IAddon;
+typedef std::shared_ptr<IAddon> AddonPtr;
+} // namespace ADDON
```

Comment:
Is this added by clang format? Why don't the others get this?

Reply:
Yes, it's a clang format setting